qrenderdoc/Windows/PipelineState/D3D11PipelineStateViewer.ui

```diff
@@ -109,6 +115,9 @@
       </item>
       <item>
        <widget class="QToolButton" name="exportHTML">
+        <property name="toolTip">
+         <string>Export</string>
```

Comment:
The tooltip can be a bit more verbose here - something like:

> Export the current pipeline state to an HTML file.

The text can be copy-pasted to the other pipeline viewers.